index.html

```diff
@@ -19,4 +31,4 @@ <h2>lorem</h2>
 cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non
 proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
 </body>
-</html>
```

Comment:
should it be having two html tags

Reply:
made the neccesary changes.. please review